packages/api/src/web5.ts

```diff
@@ -158,6 +159,8 @@ export type Web5ConnectResult = {
    * {@link WalletConnectOptions} was provided.
    */
   delegateDid?: string;
+
+  delegateGrants: DelegateGrant[];
```

Comment:
```suggestion
```

Let's remove this. We just added this as a temporary way to test and display on the requesting app something when we were pairing.

Now that the delegate sync part is in connect, we can demo it by writing a record as alice.